src/importexport/ove/internal/ove.cpp

```diff
@@ -4392,6 +4405,7 @@ bool OvscParse::parse()
     Block placeHolder;
 
     m_handle = &handle;
+    auto _ = ResetToNull(&m_handle);
```

Comment:
For some reason I don't get that to cimpile in QtCreator/MinGW:
```
C:\Users\Jojo\Documents\GitHub\MuseScore\importexport\ove\ove.cpp:3873: Error: missing template arguments before '(' token
...\importexport\ove\ove.cpp: In member function 'virtual bool OVE::TrackParse::parse()':
...\importexport\ove\ove.cpp:3873:27: error: missing template arguments before '(' token
       auto _ = ResetToNull(&handle_);
                           ^
```
```

Reply:
Changing those calls to `auto _ = ResetToNull<typeof(*handle_)>(&handle_);` fixes that

Reply:
Interesting... it looks like MinGW doesn't like [class template argument deduction](https://en.cppreference.com/w/cpp/language/class_template_argument_deduction) for some reason.

Reply:
No wonder, as this is C++17. AFAIK we're not requiring this anywhere yet (at least not for 3.x? But I thing for master neither)

Reply:
From [here](https://musescore.org/en/handbook/developers-handbook/compilation/compile-instructions-windows-visual-studio/compile-0), it looks like C++17 is required for master.
> As of 484f8dc, 09Oct2020, Qt 5.15 (and a C++17 capable toolchain) is required for the master branch.

Reply:
Regardless, I do think there's no reason to break anything for this, so I've written in a fix.

Reply:
I see. Still using `typeof` might be better